lib/matplotlib/ticker.py

```diff
@@ -432,21 +435,76 @@ def __init__(self, useOffset=None, useMathText=None, useLocale=None):
         self.orderOfMagnitude = 0
         self.format = ''
         self._scientific = True
+        self._offsetString = None
         self._powerlimits = rcParams['axes.formatter.limits']
         if useLocale is None:
             useLocale = rcParams['axes.formatter.use_locale']
         self._useLocale = useLocale
 
+    def get_useScalingFactor(self):
+        return self._usingScaling
+
+    def set_useScalingFactor(self, val):
+        """
+        Control and set the scaling factor. Disabling this returns the
+        formatter to its default behavior, e.g. it will try to find
+        an appropriate scaling/offset if enabled.
+        Note that either offset or scaling can be used. Therefore,
+        this automatically turns off offset
+
+        Parameters
+        ----------
+        val : (True|False|numeric)
+            Enable, disable, or sets and enables the use of scaling factor
+            in the axis.
+
+        Returns
+        -------
+        NONE
+        """
+
+        if isinstance(val, bool):
```

Comment:
fair enough.  Given the existing API I am convinced that this is the best of the bad options.